express-get-json/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const grades = [
+  {
+    id: 1,
+    name: 'Matt Chen',
+    course: 'Everything',
+    grade: 100
+  },
+  {
+    id: 2,
+    name: 'Rock Lee',
+    course: 'Taijustu',
+    grade: 99
+  }
+];
+const app = express();
+
+app.use((req, res, next) => {
+  if (req.originalUrl === '/api/grades') {
+    res.json(grades);
+  }
+});
```

Comment:
instead of `app.use()` we're doing `app.get()` for this exercise, as in the routing basics guide of the express docs.